Makefile

```diff
@@ -21,60 +59,41 @@ install_protoc:
 		cd .. && \
 		rm -rf protobuf-3.4.1
 
-protoc:
-	## Note to self:
-	## On "error while loading shared libraries: libprotobuf.so.14: cannot open shared object file: No such file or directory"
-	##   ldconfig (may require sudo)
-	## https://stackoverflow.com/a/25518702
-	protoc $(INCLUDE) --gogo_out=plugins=grpc:. types/*.proto
-	@ echo "--> adding nolint declarations to protobuf generated files"
-	@ awk '/package types/ { print "//nolint: gas"; print; next }1' types/types.pb.go > types/types.pb.go.new
-	@ mv types/types.pb.go.new types/types.pb.go
+update_tools:
+	@echo "--> Updating tools"
+	@go get -u $(GOTOOLS)
 
-install:
-	@ go install ./cmd/...
+get_vendor_deps:
+	@rm -rf vendor/
+	@echo "--> Running glide install"
+	@glide install
 
-build:
-	@ go build -i ./cmd/...
 
-dist:
-	@ bash scripts/dist.sh
-	@ bash scripts/publish.sh
+########################################
+### Testing
 
 test:
-	@ find . -path ./vendor -prune -o -name "*.sock" -exec rm {} \;
-	@ echo "==> Running go test"
-	@ go test $(PACKAGES)
+	@find . -path ./vendor -prune -o -name "*.sock" -exec rm {} \;
+	@echo "==> Running go test"
+	@go test $(PACKAGES)
 
 test_race:
-	@ find . -path ./vendor -prune -o -name "*.sock" -exec rm {} \;
```

Comment:
It's odd that we had a space between `@` and the `command`, thanks for fixing this!